disagg_logits_repro.py

```
@@ -0,0 +1,272 @@
+            return {"status": "ok"}
+
+        @app.post("/prefill")
+        async def prefill(req: PrefillRequest) -> PrefillResponse:
+            if not self.ready or self.llm is None:
+                return JSONResponse({"status": "not ready"}, status_code=503)
+            return await self._generate_local_prefill(req)
+
+        thread = threading.Thread(
+            target=uvicorn.run,
+            args=(app,),
+            kwargs={"host": "0.0.0.0", "port": 8000, "log_level": "info"},
```

Ruff (0.15.2)

[error] Possible binding to all interfaces

(S104)